low-level/files/opt/seagate/sspl/bin/consume_cfg_change_alert

```diff
@@ -23,7 +23,7 @@ SCRIPT_DIR=$(dirname $0)
 exec &>> /var/log/$PRODUCT_FAMILY/sspl/consul-${0##*/}.log
 exec &> >(stdbuf -oL gawk '{ print strftime("%Y-%m-%d %H:%M:%S"), $0 }')
 
-SSPL_PID=`/sbin/pidof -s /usr/bin/sspl_ll_d`
+SSPL_PID=`/usr/bin/pgrep -d " " -f /usr/bin/sspl_ll_d`
```

Comment:
For some reason, what if multiple PIDs comes as o/p here, possibly because of some zombie sspl process. 
pgrep -c (to check process count) and -n(to consider latest process id) options would be helpful here to choose the intended PID, in worst cases

Reply:
>>> For some reason, what if multiple PIDs come as o/p here, possibly because of some zombie sspl process.

The above syntax will return space-separated PID's. eg. 1234 2233 1234 etc.

Reply:
yes, you need to pick only latest one am saying

Reply:
What if I kill all the PIDs which is related to the sspl_ll_d service?